web/src/components/DandisetsPage.vue

```diff
@@ -51,11 +51,14 @@
         >
           {{ option.name }}
           <v-icon right>
-            <template v-if="sortDir === 1 || sortOption !== i">
-              mdi-sort-ascending
+            <!-- For some reason, MDI's icons are represented backwards of what you'd expect.
+             Descending has an arrow pointing upwards, which is usually meant to represent ascending.
+             We intentionally switch the values here, this is not a bug. -->
```

Comment:
I think we can get rid of the comment now. I know the variant icons still have a strange naming, but I think visually they make sense.

```suggestion
```